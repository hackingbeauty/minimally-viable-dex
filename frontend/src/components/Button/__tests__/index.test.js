import React from 'react'
import { render } from 'tests/test-utils'
import { cleanup } from '@testing-library/react'
import Button from 'components/AppBar'

afterEach(cleanup)

test('renders correctly', () => {
  const { container } = render(<Button />)
  expect(container.firstChild).toMatchSnapshot()
})
