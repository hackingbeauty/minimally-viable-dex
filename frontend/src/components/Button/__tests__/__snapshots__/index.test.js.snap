// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="styles"
>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary app-bar mui-fixed MuiPaper-elevation4"
  />
</div>
`;
